python/lsst/validate/drp/matchreduce.py

```diff
@@ -190,6 +194,17 @@ def _loadAndMatchCatalogs(repo, dataIds, matchRadius,
                                        'PSF magnitude'))
     mapper.addOutputField(Field[float]('base_PsfFlux_magErr',
                                        'PSF magnitude uncertainty'))
+    if not skipNonSrd:
+        # Needed because addOutputField(... 'slot_ModelFlux_mag') will add a field with that literal name
```

Comment:
" Need to check for column existence through an alias because `addOutputField(...`  ..."